gnucash/import-export/csv-exp/csv-transactions-export.cpp

```diff
@@ -158,12 +158,26 @@ get_commodity (Transaction *trans)
     return gnc_commodity_get_unique_name (xaccTransGetCurrency (trans));
 }
 
+static GNCPrintAmountInfo number_pinfo =
+{
+    .commodity = nullptr,
+    .max_decimal_places = 8,
+    .min_decimal_places = 0,
+    .use_separators = 0,
+    .use_symbol = 0,
+    .use_locale = 0,
+    .monetary = 0,
+    .force_fit = 0,
+    .round = 0
+};
+
 // Amount with Symbol or not
 static std::string
 get_amount (Split *split, bool t_void, bool symbol)
 {
     auto amt_num{t_void ? xaccSplitVoidFormerAmount (split) : xaccSplitGetAmount (split)};
-    return xaccPrintAmount (amt_num, gnc_split_amount_print_info (split, symbol));
+    auto pinfo{symbol ? gnc_split_amount_print_info (split, symbol) : number_pinfo};
+    return xaccPrintAmount (amt_num, pinfo);
```

Comment:
ISTM 
```
auto pinfo{gnc_split_amount_print_info(split, symbol)};
if (!symbol)
    pinfo.use_separators = FALSE;
```
rather than a generic `number_pinfo`.